core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -621,8 +621,10 @@ void openToc() {
       compatCallback.finish();
     } else if (drawerLayout.isDrawerOpen(GravityCompat.END)) {
       drawerLayout.closeDrawers();
-    } else if (getCurrentWebView().canGoBack()) {
+    } else if (getCurrentWebView().canGoBack() && !HOME_URL.equals(getCurrentWebView().getUrl())) {
```

Comment:
I think you can remove this condition and everything will continue working

Reply:
Actually, that condition is required. Without it, the app is stuck in some loop. 
Here is the video if that line is removed
[Link](https://imgur.com/Px0S11V)